src/site/content/en/blog/mailru-cwv/index.md

```diff
@@ -0,0 +1,290 @@
+ 
+Svelte implements reactivity in a way that [doesn't use Virtual DOM](https://svelte.dev/blog/virtual-dom-is-pure-overhead), which makes it less resource-intensive. [Svelte's approach removes unused functions from production bundles](https://svelte.dev/blog/frameworks-without-the-framework) because the code implementing them isn't generated by the compiler if the functions aren't used. Unused code is removed during compilation, resulting in smaller bundles.
+ 
+Reduced bundle sizes may positively affect metrics such as [Total Blocking Time (TBT)](https://web.dev/tbt/). However, the results of our CWV optimization efforts may be compelling to those using popular frontend JavaScript frameworks.
+
+## Tracking performance metrics
+
+Before optimizing CWV, it's helpful to [evaluate performance in the field](https://web.dev/vitals-field-measurement-best-practices/). Prior to the introduction of CWV, other metrics such as [First Contentful Paint (FCP)](https://web.dev/fcp/) were monitored and presented in an internal performance dashboard.
+
+The metrics collection mechanism was modified to collect CWV values and represent them in our performance dashboard. In line with Google's recommendations, our performance tracking system uses [PerformanceObserver API](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceObserver) to obtain metrics, which is part of [our universal frontend "Platform"](https://github.com/mail-core) inside Mail.ru.
+
+The dashboard's CWV monitoring showed the following metrics for users (mean values for the week of 15–21 March 2021):
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th colspan="2" width="40%">Metrics group name</th>
+        <th colspan="3" width="30%">CWV</th>
+        <th colspan="3" width="30%">Other Web Vitals</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td colspan="2">Metric name</td>
+        <td>LCP</td>
+        <td>FID</td>
+        <td>CLS</td>
+        <td>FCP</td>
+        <td>TBT</td>
+        <td>TTI</td>
+      </tr>
+      <tr>
+        <td rowspan="3">Share of users in accordance with CWV thresholds</td>
+        <td>good</td>
+        <td>52%</td>
+        <td>92%</td>
+        <td>33%</td>
+        <td>35%</td>
+        <td>42%</td>
+        <td>43%</td>
+      </tr>
+      <tr>
+        <td>needs-improvement</td>
+        <td>19%</td>
+        <td>5%</td>
+        <td>23%</td>
+        <td>38%</td>
+        <td>16%</td>
+        <td>25%</td>
+      </tr>
+      <tr>
+        <td>poor</td>
+        <td>29%</td>
+        <td>3%</td>
+        <td>44%</td>
+        <td>27%</td>
+        <td>42%</td>
+        <td>32%</td>
+      </tr>
+    </tbody>
+    <caption>CWV metrics for the week of 15–21 March 2021</caption>
+  </table>
+</div>
+ 
+<figure class="w-figure">
+  {% Img src="image/dB6B4Sr8kaaT0KZujRBFC303oFR2/ZL7y3ZCpbMcqlOv9mLpV.png", alt="Web Vitals values before the improvements.", width="800", height="479" %}
```

Comment:
This is _really_ hard to do in alt text with these types of images, but anything you can do to make the alt text more descriptive (while not running _too_ long) will help.